searx/settings.yml

```diff
@@ -1548,6 +1548,24 @@ engines:
     page_size: 25
     disabled: true
 
+  - name: right dao
+    engine: xpath
+    paging: true
+    page_size: 12
+    search_url: https://rightdao.com/search?q={query}&start={pageno}
+    results_xpath: //div[@class="description"]
```

Comment:
Why don't you use `//div[contains(@class, "item")`? That way you wouldn't need to use `../` below, would you?

Reply:
The class item is also used for headers and news stories in the results. Only genuine results have a description.